float64 rolltargetvalue
bool rolltargetoverride
float64 pitchtargetvalue
bool pitchtargetoverride
float64 velocitytargetvalue
bool velocitytargetoverride
float64 climbratetargetvalue
bool climbratetargetoverride
float64 yawratetargetvalue
bool yawratetargetoverride
float64 rolloutputvalue
bool rolloutputoverride
float64 pitchoutputvalue
bool pitchoutputoverride
float64 yawoutputvalue
bool yawoutputoverride
float64 throttleoutputvalue
bool throttleoutputoverride
float64 flapoutputvalue
bool flapoutputoverride

bool activate
bool overridemaneuverplanner
---
bool valid_request